analyzers/its/expected/akka.net/S3900-Akka.Cluster.TestKit-netstandard2.0.json

```diff
@@ -1,11 +1,5 @@
 {
   "Issues": [
-    {
-      "Id": "S3900",
-      "Message": "Refactor this method to add validation of parameter \u0027roles\u0027 before using it.",
-      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/akka.net/src/core/Akka.Cluster.TestKit/MultiNodeClusterSpec.cs#L269",
```

Comment:
[Link](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/akka.net/src/core/Akka.Cluster.TestKit/MultiNodeClusterSpec.cs#L269)
`NotNull` learnt multiple times, the first being `roles.First()` on line 262.